docs/jupyter/t_pipelines/t_icp_registration.ipynb

```diff
@@ -405,7 +405,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 1. Set Inputs and Parameters"
+    "### 1. Set Inputs and Parameters"
```

Comment:
More info on why here - https://github.com/spatialaudio/nbsphinx/issues/373#issuecomment-572739075

P.S. The changes here are getting over written somehow when building docs.

Reply:
Found my oversight. So, changes here are getting ignored because a pre-built `t_icp_registration` notebook file is getting downloaded in `make_docs.py` due to https://github.com/isl-org/Open3D/pull/4321.

Is the documentation still built without cuda on CI/CD?

Reply:
I think so. It is safe to keep the changes here now, we can study the best option regarding large notebook files in another PR.